source/partials/_header.html.erb

```diff
@@ -1,4 +1,4 @@
 <header class="site-header content-wrapper" role="banner">
-  <h1>apprentice.io</h1>
+  <h1>apprentice.thoughtbot.com</h1>
```

Comment:
How about changing this to `Community resources for software apprenticeships` and dropping the following paragraph entirely?

Reply:
I like it. My only hesitation is that it's a little long. Having said that - it's exactly what this is, and I'd like to go with _something_ to replace apprentice.io. I'll make the change.